client/post-editor/editor-ground-control/index.jsx

```diff
@@ -220,9 +227,14 @@ export class EditorGroundControl extends PureComponent {
 	}
 
 	onBackButtonClick = () => {
+		this.props.recordTracksEvent( 'calypso_editor_back_button_click' );
```

Comment:
Exposing these low-level functions inside of React components is definitely something we like to avoid on account of the fact that the UI usually doesn't really care itself about tracking metrics. It would maybe be more meaningful to pass in some kind of `onBackButtonClick` from `connect()` than it would be to pass in `recordTracksEvent()`

That we are tracking these buttons may be incidental to the fact that we are providing a click handler. In some cases, in very few very rare cases, we may actually care about tracking the button click (maybe this is one of them)? then it's fair to pass in some function from `connect()` that's paramount to `trackClick()` but even in those cases we can leave `recordTracksEvent()` out of it - it's a tool but not itself a part of a component's API

Reply:
Thanks for your thoughts! Is this better practice? https://github.com/Automattic/wp-calypso/pull/20797/commits/6d2313490b9e775728976ae0ef127232daa99e8f

Reply:
Is what you're saying here, pass the `onClick` event and track in there, instead of creating handlers that just track and call the passed prop?